README.md

```diff
@@ -125,23 +86,25 @@ ya29.zyxwvutsrqpnmolkjihgfedcba
 The same as `fetch`, except the output is in HTTP header format:
 
 ```bash
-$ oauth2l header userinfo.email
+$ oauth2l header --scope cloud-platform
 Authorization: Bearer ya29.zyxwvutsrqpnmolkjihgfedcba
 ```
 
-The `header` command is designed to be easy to use with `curl`. For example,
-the following command uses the PubSub API to list all PubSub topics.
+The `header` command is designed to be easy to use with the `curl` CLI. For
```

Comment:
I think it is still useful to retain "header" for two reasons: One, for educational purposes, it would be good to see what the header looks like. Two, users may want to use something else besides curl for test requests, such as the Postman plug-in.